qmi/instruments/teraxion/tfn.py

```diff
@@ -399,8 +414,8 @@ def get_manufacturer_name(self) -> str:
         resp = self._read(Teraxion_TFNCommand_GetManufacturerName)
         time.sleep(self.GET_LONG_PROCESS_TIME)
         # get the data after the status bytes and ignore the null bytes
-        manufacturer_name = resp[self.LEN_STATUS_BYTES:]
-        return manufacturer_name.decode("ascii")[:manufacturer_name.find(b"\x00")]
+        manufacturer_name = resp[self.LEN_STATUS_BYTES :]
+        return manufacturer_name.decode("ascii")[: manufacturer_name.find(b"\x00")]
```

Comment:
Minor: These spaces around the `:` look a bit ugly. Is it Black that's doing that? Anyhow, a preference thing I suppose.